dbms/src/Storages/KVStore/FFI/ProxyFFI.cpp

```diff
@@ -1009,11 +1010,7 @@ BaseBuffView GetLockByKey(const EngineStoreServerWrap * server, uint64_t region_
         if (!value)
         {
             // key not exist
-            LOG_WARNING(
-                Logger::get(),
-                "Failed to get lock by key {}, region_id={}",
-                tikv_key.toDebugString(),
-                region_id);
+            LOG_DEBUG(Logger::get(), "Failed to get lock by key {}, region_id={}", tikv_key.toDebugString(), region_id);
```

Comment:
Why change this to DEBUG?

Reply:
it has been changed to debug level in the cse branch because there could be lots of logging after file base large txn